modules/customize-certificates-manually-rotate-service-ca.adoc

```diff
@@ -5,8 +5,14 @@
 [id="manually-rotate-service-ca_{context}"]
 = Manually rotate the service CA certificate
 
-The service CA is valid for one year after {product-title} is
-installed. Follow these steps to manually refresh the service CA before the expiration date.
+The service CA is valid for 14 months and is automatically refreshed when there is less than six months validity left.
+
+If necessary, you can manually refresh the service CA using the following procedure.
+
+[WARNING]
+====
+Manual rotation does not maintain trust with the previous service CA. This might result in service disruption until all affected services are refreshed to use the new service CA.
```

Comment:
I think it might be clearer if the focus was more on the trust between the CAs instead of the rotation process. I might say something like:
> A manually-rotated service CA does not maintain trust with the previous service CA. You might experience a temporary service disruption until the Pods in the server restart, which applies the new service CA to the services.

Reply:
Note that the service CA 'issues' service serving certificates. The last sentence would therefore more correctly be `until the pods in the cluster are restarted, which ensures that pods are using service serving certificates issued by the new CA`.